ui/pages/groups/[id].js

```diff
@@ -120,8 +123,30 @@ export default function GroupDetails() {
   ]
 
   // Don't allow deleting the last group
-  const hideRemoveGroupBtn =
-    !isAdmin || (infraAdmins?.length === 1 && adminGroups.includes(group?.id))
+  const showRemoveGroupBtn =
+    isAdmin && !(infraAdmins?.length === 1 && adminGroups.includes(group?.id))
```

Comment:
I believe you can simplify this to:

```suggestion
    isAdmin && !adminGroups.includes(group?.id)
```

Reply:
no we don't want to remove the check for number of infraAdmins - Because we show the remove group btn as long as there is more than one infra admin. We want to hide the btn when there is only one infra admin and that infra admin is the current group

Reply:
That makes sense. Shortening the expression (if we can), or at least making it clearer would make this easier to understand. One idea is to break this into two – make this `isLastAdminGroup` and then use `isAdmin` below inline, or something similar.